packages/example-contracts/contracts/shared/ZetaEthMock.sol

```diff
@@ -5,6 +5,8 @@ import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/utils/Context.sol";
 
+import "@zetachain/protocol-contracts/contracts/ZetaTokenConsumerUniV2.strategy.sol";
+
```

Comment:
Do we need this?

Reply:
move to another file. was there because the test contract for imports (typechain) has a different compiler. just create a new file to this import

Reply:
Cool 👍